src/fckit/module/fckit_configuration.cc

```diff
@@ -275,7 +275,7 @@ int32 c_fckit_configuration_get_string( const Configuration* This, const char* n
         return false;
     }
     size  = s.size() + 1;
-    value = new char[size];
+    value = new char[size + 1];
```

Comment:
Surprising as size was already `s.size() + 1` . You sure about this one?

Reply:
You are right, this one is not necessary. I did a grep for `char[size]` but I didn't notice that size was already `s.size()+1`. I just removed it.